packages/client/src/generation/utils.ts

```diff
@@ -329,3 +329,12 @@ export function unique<T>(arr: T[]): T[] {
 
   return result
 }
+
+/**
+ *
+ * @param paths - list of paths in posix or win32 format to join and then
+ *     replace all backslash separators with a forward slash
+ */
+export function makePosixImportPath(...paths: string[]): string {
+  return path.join(...paths.map((segment) => segment.replace(/\\$/, ''))).replace(/\\/g, path.posix.sep)
```

Comment:
Not sure how much it's practically important here, but generally on Unix a backslash may be a valid character in a file name (although it would break things when embedded into a string without proper escaping nevertheless).

Reply:
I would probably prefer to replace `path.sep` rather than hardcoding `\\`, but I understand that it may make testing this function more complicated.

Reply:
I made a few more changes based on your feedback. I added the prefix `'./'` for when the runtime would be in a child directory because `path.join` automatically removes the prefix and `require` will go looking in the `node_modules`. I also only replace the back slash on Windows in case it is a part of the file name on Linux.